src/helpers/articles.js

```diff
@@ -56,7 +70,7 @@ class ArticleModel {
   /**
    * @description - This method is responsible for quering the database for an article
    * @static
-   * @param {object} request - Request sent to the router
+   * @param {object} slug - Request sent to the middleware
```

Comment:
@marcdomain, nice implementation and I like the fact that you fixed the JSDoc comment issues on the `article` class